### ISSUE TITLE
PHP Warnings

### ISSUE THREAD

**Issue by meshkov** [modxcms/revolution#13041]
### Summary

Having the following errors:
[Tue May 31 15:30:51 2016] [error] [client xxx.xxx.xxx.xxx] PHP Warning:  session_start(): Session callback expects true/false return value in /home/x/xxx/xxx.xx/public_html/core/model/modx/modx.class.php on line 2270, referer: http://xxx.xx/manager/?a=resource/update&id=55
### Step to reproduce

I update the resources and when i try to save resource i have this errors and error in browser ERR_CONNECTION_REFUSED.
### Environment

MODX Revolution 2.5.0-pl (traditional)
PHP Version 7.0.6
Google Chrome 51.0.2704.63 m

**Comment by OptimusCrime** [comment]
This was first reported in #12797.

**Comment by christianseel** [comment]
Are you sure that you're using 2.5.0-pl here? I don't see this error when running MODX 2.5.0-pl with PHP 7.0.6

**Comment by meshkov** [comment]
I use MODX Revolution 2.5.0-pl (traditional).

**Comment by meshkov** [comment]
![2016-06-02_15-11-50](https://cloud.githubusercontent.com/assets/1703670/15744409/7ad9918a-28d4-11e6-9506-8e1f24897916.png)
After that error - i have error in browser ERR_CONNECTION_REFUSED. And have errors in logs.

**Comment by OptimusCrime** [comment]
Looks like this is an error with your server or your setup. I do not have the same problems here.

**Comment by Realetive** [comment]
What about repeat this actions for request this error in other browsers?

**Comment by meshkov** [comment]
In mac os with google chrome i don`t have errors

**Comment by OptimusCrime** [comment]
What browser produces the error?

**Comment by OptimusCrime** [comment]
Closing this due to lack of feedback. It was also reported that the identical version did not produce the same error in another instance. Feel free to reopen this issue if you still experience this problem or if you can give more information about the incident.

@modxbot close

**Comment by Jako** [comment]
This occurs on an 2.5.2 installation with PHP 7.0.14 on the first load of the site (when a new session is initialized). It is just a warning, but it occurs in the frontend output on that installation.

**Comment by sottwell** [comment]
Is this only when logging in, or for first site load with (anonymous) session?

I'm using PHP 7.1.0 with Revo 2.5.2 on a MAMP stack with no problems.

**Comment by Jako** [comment]
I have to ask my client for this. I think he has loaded the page without any login attempt at home.

**Comment by Jako** [comment]
![screen](https://cloud.githubusercontent.com/assets/148371/22163701/32587ebc-df54-11e6-9dbf-82850b4f986d.png)

**Comment by sottwell** [comment]
Does the same thing happen when loading the Manager login page?

**Comment by Jako** [comment]
We will check this. At the moment i have cleared the modx_session table on that installation for a try.

**Comment by OptimusCrime** [comment]
Running MODX 2.5.4-pl and PHP 7.0.14-1~dotdeb+8.1. No warnings here. Could you try to upgrade to the latest stable?

**Comment by OptimusCrime** [comment]
Still unable to reproduce this here. My setup is listed in the previous comment. Can someone else reproduce this?

#modxbughunt

**Comment by meshkov** [comment]
I have this php warnings.
i wrote in another issues:
https://github.com/modxcms/revolution/issues/13073

**Comment by meshkov** [comment]
i think problem with session_set_save_handler.
https://secure.php.net/manual/en/function.session-set-save-handler.php

**Comment by meshkov** [comment]
This problem only on php 7.
From this issue https://github.com/modxcms/revolution/issues/12797 (PHP7 Compatibility Checker):
```
> Line 2411: Check that callbacks that are passed to "session_set_save_handler" and return false or -1 (if any) operate correctly
    session_set_save_handler(array(&$sh, 'open'), array(&$sh, 'close'), array(&$sh, 'read'), array(&$sh, 'write'), array(&$sh, 'destroy'), array(&$sh, 'gc'));
```

**Comment by Mark-H** [comment]
Fixed in 2.6
